docs/admins/guide.rst

```diff
@@ -0,0 +1,173 @@
+
+A good way to approach this step is to think of a practical, real-world setting that best demonstrates the concept being taught.
+
+.. admonition:: Example
+
+   A good real-world example for Decimals is prices in dollars and cents. A story emanating from this idea might be a young girl in a shop grappling with a price as she tries to relate it to decimal concepts. This setting might then span several lessons as the character learns how to read the price, connects the tenths and hundredths place values to dimes and pennies, compares decimal place values and so on.
+
+Your high-level story outline will include:
+
+ * A brief overview of the characters, settings and premise.
+ * One to two paragraphs describing what happens in each lesson.
+
+.. _skilltable:
+
+Creating a Skill Table
+-----------------------
+
+The Skill Table is where you will build upon the previous two steps and further elaborate the skills you have identified. Before you create the table however, list the Prerequisite skills, Acquired skills and Practiced skills for the lesson.
+
+ * **Prerequisite skills**: Skills the learner would need before starting the lesson.
+ * **Acquired skills**: Skills the learner will acquire by the end of the lesson.
+ * **Practiced skills**: Skills that will be taught and practiced during the lesson.
+
+
+The skill table should look like this:
+
+.. figure:: /images/skill_table_format.png
+   :alt: Skill Table format
+   :align: center
+
+   *Skill Table*
+
+
+.. topic:: Explanation of columns in Skill Table
+
+    * **Skill**: The skills you list here are the same as in the :ref:`Skill Tree <skilltree>`. Start with a target question in mind and work backwards to determine what skills students need to answer it. The more more detailed you can get while working backwards step-by-step, the more skills you will be able to identify. Be aware of potential 'hidden' skills that aren't taught explicitly in the lessons.
+
+    * **Specific Errors**: The description of misconceptions should be really specific, such that anyone should be able to read the misconception and predict how a student will answer given a question. In other words, it is insufficient to just say, "Student cannot do skill X correctly."
+
+    * **Remediation**: Some things you might list here include "Return to card that (addresses the misconception)." or "Show a picture that (clarifies a misunderstanding)."
+
+    * **Development of the Topic**: This is the precursor to your script and the most important column. The questions you list here will appear in the :ref:`script <script>`. Make sure each question only introduces *one* new skill to ensure you aren't making leaps too big for the learner's understanding.
```

Comment:
Does 'script' mean the exploration?

Reply:
Yes, here a script means a draft of what the lesson will look like in Oppia.